launcher/src/Instance.scala

```diff
@@ -103,16 +106,43 @@ object Instance{
             case _ => ???
           }
         case multiple =>
-          throw new Exception(
-            s"More than one running instance found: ${multiple.map(_.instanceId()).mkString(", ")}. You can delete extra instances in the aws-dev-admin console (double check your region!)."
-          )
+          val instanceOptions = multiple.zipWithIndex.map(t => {
+            val (instance, idx) = t
```

Comment:
nit: These two lines can be collapsed into 

```
multiple.zipWithIndex.map { case (instance, idx) =>
...
}
```

In Scala round parenthesis and braces are pretty much equivalent, at least around lambdas. The syntax

```
{
  case Foo(x, y) => ..
  case ..
}
```
is called a `PartialFunction` and is a subtype of function, so it can be passed instead of a lambda for map (or any other higher-order function)